tests/components/ct-react-vite/src/App.spec.tsx

```diff
@@ -18,3 +18,10 @@ test('should configure app', async ({ page, mount }) => {
   });
   expect(messages).toEqual(['Before mount: {\"route\":\"A\"}', 'After mount']);
 });
+
+test('should unmount', async ({ page, mount, unmount }) => {
+  const component = await mount(<App></App>);
+  await expect(page.locator('#root')).toContainText('Hello Vite + React!');
+  unmount(component);
```

Comment:
should this be awaited? the `await expect(page.locator('#root'))` implicitly waits for this. I think this can be confusing